csunplugged/templates/base.html

```diff
@@ -53,63 +71,51 @@
-              </g>
-            </svg>
-          </button>
-          <div id="navbarLanguageDropdown" class="dropdown-menu dropdown-menu-{{ LANGUAGE_END }}" aria-labelledby="navbarLanguageSelector">
-            {% for language in LANGUAGES %}
-              {% get_language_info for language.0 as lang %}
-              <a class="dropdown-item{% if LANGUAGE_CODE == lang.code %} active{% endif %}" href="{% translate_url language.0 %}">{{ lang.name_local|capfirst }}</a>
-            {% endfor %}
-          </div>
-        </div>
         <div class="collapse navbar-collapse" id="navbarNav">
           <div class="navbar-nav navbar-nav-i18n mr-auto">
             <a class="nav-item nav-link" href="{% url 'topics:index' %}">{% trans "Topics" %}</a>
             <a class="nav-item nav-link" href="{% url 'resources:index' %}">{% trans "Printables" %}</a>
             <a class="nav-item nav-link" href="{% url 'general:about' %}">{% trans "About" %}</a>
           </div>
-          {% if LANGUAGE_CODE == "en" %}
-            <!-- Search is only currently available in English, see https://github.com/uccser/cs-unplugged/issues/989 -->
-            <form method="get" action="{% url 'search:index' %}">
-              <div class="input-group" id="search-navbar">
-                <input type="text" class="form-control form-control-sm border-0" placeholder="Search" aria-label="Search" name="q">
-                <div class="input-group-append">
-                  <button class="btn btn-outline-light btn-sm" type="submit">
-                    <!-- SVG from Icomoon-Free -->
-                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32" class="icon icon-search">
-                      <title>{% trans "Search" %}</title>
-                      <path d="M31.008 27.231l-7.58-6.447c-0.784-0.705-1.622-1.029-2.299-0.998 1.789-2.096 2.87-4.815 2.87-7.787 0-6.627-5.373-12-12-12s-12 5.373-12 12 5.373 12 12 12c2.972 0 5.691-1.081 7.787-2.87-0.031 0.677 0.293 1.515 0.998 2.299l6.447 7.58c1.104 1.226 2.907 1.33 4.007 0.23s0.997-2.903-0.23-4.007zM12 20c-4.418 0-8-3.582-8-8s3.582-8 8-8 8 3.582 8 8-3.582 8-8 8z"></path>
-                    </svg>
-                  </button>
+          <form method="get" action="{% url 'search:index' %}">
```

Comment:
For some reason this section is overriding the code on `develop`, which should not be overridden.